cyanobyte/codegen.py

```diff
@@ -21,14 +21,21 @@
 import click
 from yaml import load
 
+import argparse
+from os import path
+import subprocess
+import json
+
+from convert_json_to_yaml import convert_json_to_yaml
+
 try:
     from yaml import CLoader as Loader
 except ImportError:
     from yaml import Loader
 from jinja2 import Environment, FileSystemLoader
 
 _VERSION = "0.1.0"
-_DEBUG = False
+_DEBUG = True
```

Comment:
Can we keep this as `False`. It can be set using the `--debug true` on the command line